etc/chromeos/systemd-resolve-dns.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# WARNING: This is not idempotent (yet). Only run this once
+
+# check systemd version
+SYSTEMD_VERSION=$(systemctl --version | head --lines 1 | awk '{ print $2}')
+if [[ $SYSTEMD_VERSION -lt 246 ]]; then
+    echo "Systemd version must be 246+ in order to enable consul dns."
+    echo "Please update your chromeos container to Debian Bullseye by running"
+    # shellcheck disable=SC2016
+    echo '`sudo bash /opt/google/cros-containers/bin/upgrade_container DEBIAN_BULLSEYE`'
+fi
+
+# Since systemd-resolved listens on 127.0.0.53, we're going to also have it listen on the docker0 bridge.
+# This will usually resolve to 172.17.0.1
+DOCKER0_BRIDGE=$(docker network inspect bridge --format='{{(index .IPAM.Config 0).Gateway}}')
+echo "DNSStubListenerExtra=$DOCKER0_BRIDGE" | sudo tee --append /etc/systemd/resolved.conf
```

Comment:
This isn't necessary, right? We just need `/etc/systemd/resolved.conf.d/consul.conf`.

Messing with the main config, and even having `/etc/systemd/resolved.conf.d/dns_servers.conf`, seems unnecessary and error prone.